_pages/plugins/Fast4DReg.md

```diff
@@ -0,0 +1,180 @@
+- **Crop output:** Crop output will crop out the black frame created by the image moving. This will be performed on default if continued to z-correction.
+- **z-drift correction:** If you want to correct for z-drift, tick the z-drift correction box.
+- **Reslice mode:** Reslice mode lets you decide if you want to create the projection along the x-axis (top) or y-axis (left).
+- **Projection type:** Select the projection type used for z-drift estimation (maximum or average intensity) 
+- **Extend stack to fit:** Extend stack to fit will create extra slices to the stack to ensure that the whole stack is saved.
+- **Save RAM:** If ticked the z-corrected image is built frame by frame instead of building the image in one go. This saves RAM but approximately doubles the time for processing.
+  
+3. Click ok. The script will run.
+4. When the script has completed the process, you will have the following files in a new folder:
+       - corrected images
+       - drift plots
+       - drift tables
+       - a settings file, you can use to run the script on another image with identical parameters. 
+   
+   The folder will have an unique identifyer: *fileName + date + experiment number*. 
+   If you plan to apply the correction to another channel, make sure not to move these files to another folder.
+  
+### Apply drift
+
+1. Open the "apply" script and click run. The user interface opens.  
+
+![image](../../media/Fast4DReg-applyUI.png)
+*Figure 3: Apply user interface*
+
+
+- **Select the path to the file to be corrected:** Navigate to your image to be corrected here.
+- **Select the settings file (csv.):**. Navigate to your settings file (called settings.csv).
+
+2. Click ok. The corrected image will be saved to the same folder with the settings file.
+
+Done!
+
+
+
+# Known issues
+
+**Fiji crashing without notice**
+
+Windows might not have OpenCL installed. If you experience Fiji crashing without notice, [install the OpenCL through the Microsoft store library.](https://www.microsoft.com/en-us/p/opencl-and-opengl-compatibility-pack/9nqpsl29bfff?activetab=pivot:overviewtab)
+
+**Importance of file locations**
+
+Make sure not to move the drift table from the results folder as the path to the drift table is hardcoded to the settings.csv file. 
+
+**Result images are black**
+
+Try disabling time averaging (1).
+
+
+# Contributors
+
+* [Joanna Pylvänäinen](https://twitter.com/JwPylvanainen)
+* [Romain F. Laine](https://twitter.com/LaineBioImaging)
+* [Guillaume Jacquemet](https://twitter.com/guijacquemet)
+
+
+
+# When using this script, please cite the NanoJ paper
+
+Laine, R. F., Tosheva, K. L., Gustafsson, N., Gray, R., Almada, P., Albrecht, D., Risa, G. T., Hurtig, F., Lindås, A. C., Baum, B., Mercer, J., Leterrier, C., Pereira, P. M., Culley, S., & Henriques, R. (2019). NanoJ: a high-performance open-source super-resolution microscopy toolbox. Journal of physics D: Applied physics, 52(16), 163001. https://doi.org/10.1088/1361-6463/ab0261
```

Comment:
If you put `doi: 10.1088/1361-6463/ab0261` in the front matter at the top, you can use `{% include citation %}` here instead of typing out the citation by hand. And other pages will be able to cite Fast4DReg easily by writing `{% include citation id='plugins/fast4dreg' %}`. See the [Citations documentation](https://imagej.net/editing/citations).